.chloggen/remove_httpforwarder.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: httpforwarder
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove extension named httpforwarder, use httpforwarderextension instead.
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [24171]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
+
+# If your change doesn't affect end users or the exported elements of any package,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+# Optional: The change log or logs in which this entry should be included.
+# e.g. '[user]' or '[user, api]'
+# Include 'user' if the change is relevant to end users.
+# Include 'api' if there is a change to a library API.
+# Default: '[user]'
+change_logs: [api, user]
```

Comment:
Why `user`? I'd expect it to be a no-op change for end users

Reply:
> Why `user`? I'd expect it to be a no-op change for end users

That's a good point. The configuration does not change. Unless we treat distro builders as users? they need to update their ocb manifests, right?

Reply:
right, what Andrzej said is probably what gave me a bit of thinking in adding to the user changelog. I don't have a strong opinion about it, I probably thought more is better than missing to mention it.

Reply:
Ok, sounds good